src/BTVNanoCommissioning/workflows/__init__.py

```diff
@@ -0,0 +1,55 @@
+ 
+# Validation
+from BTVNanoCommissioning.workflows.validation import NanoProcessor as ValidationProcessor
+
+# TTbar
+from BTVNanoCommissioning.workflows.ttbar_validation import NanoProcessor as TTbarValidProcessor
+from BTVNanoCommissioning.workflows.ttsemilep_valid_jec import NanoProcessor as TTsemilepValidJECProcessor
+from BTVNanoCommissioning.workflows.ttdilep_valid_jec import NanoProcessor as TTdilepValidJECProcessor
+from BTVNanoCommissioning.workflows.ttsemilep_valid_sf import NanoProcessor as TTsemilepValidSFProcessor
+from BTVNanoCommissioning.workflows.ttdilep_valid_sf import NanoProcessor as TTdilepValidSFProcessor
+
+# C-tag
+from BTVNanoCommissioning.workflows.ctag_emdileptt_valid_sf import NanoProcessor as CTAGEMDilepttValidSFProcessor
+from BTVNanoCommissioning.workflows.ctag_dileptt_valid_sf import NanoProcessor as CTAGDilepttValidSFProcessor
+from BTVNanoCommissioning.workflows.ctag_eDY_valid_sf import NanoProcessor as CTAGeDYValidSFProcessor
+from BTVNanoCommissioning.workflows.ctag_eWc_valid_sf import NanoProcessor as CTAGeWcValidSFProcessor 
+from BTVNanoCommissioning.workflows.ctag_Wc_valid_sf import NanoProcessor as CTAGWcValidSFProcessor
+from BTVNanoCommissioning.workflows.ctag_DY_valid_sf import NanoProcessor as CTAGDYValidSFProcessor
+from BTVNanoCommissioning.workflows.ctag_edileptt_valid_sf import NanoProcessor as CTAGEDilepttValidSFProcessor
+from BTVNanoCommissioning.workflows.ctag_ettsemilep_valid_sf import NanoProcessor as CTAGETTSemilepValidSFProcessor
+from BTVNanoCommissioning.workflows.ctag_valid_jec import NanoProcessor as CTAGValidJECProcessor
+from BTVNanoCommissioning.workflows.ctag_semileptt_valid_sf import NanoProcessor as CTAGSemilepttValidSFProcessor
+
+# FIXME - make names more systematic?
+workflows = {}
+workflows["validation"] = ValidationProcessor
+
```

Comment:
Please annotate the workflows a little

Reply:
I try to make it as `reval_jetptid` as it only contains the simplest criteria